### DISCUSSION TITLE
Is it possible to deploy a standalone kafka bridge?

### DISCUSSION THREAD

**Discussion by JaeHyeokLee** [strimzi/strimzi-kafka-operator#4371] [Q&A]
Hi, I'm trying to deploy kafka bridge without Kafka Cluster operator, but there isn's any mention about standalone Kafka Bridge at strimzi documentation.
Can I use Kafka Bridge with existing Kafka cluster that is not operated by Strimzi Kafka Cluster operator?

**Comment by scholzj** [comment]
I think there are two separate things which need to be distinguished:
1) Can I run the Strimzi Bridge using the Strimzi operator, but connected to a Kafka cluster not operated by Strimzi.
2) Can I run the Strimzi Bridge without the Strimzi operator.

The answer to 1) is yes. You can do it. You can just follow the regular documentation, configure the bootstrap server of your Kafka cluster, configure TLS, authentication etc. It does not really differ from running it against Strimzi Kafka cluster.

The answer to 2) is yes as well. But ... the main focus of the Strimzi project is to make running Apache Kafka and the related components easy using the operator pattern. You can for sure run the bridge without the operator. But it is not necessarily the way we think it should be used. That is also why nobody had time to document it for example. So yes, you can for sure use it that way, but you might need to figure out your own deployment etc.

**Reply by JaeHyeokLee** [reply]
@scholzj I installed Strimzi Kafka operator and using topic operator / kafka connect. I am trying to deploy Kafka Bridge using Strimzi Kafka Cluster operator crd. Then how can I add some annotation to k8s service that automatically generated by Kafka Bridge crd? I want to expose Kafka Bridge outside k8s cluster. In docs, load balancer service is needed to communicate. Should I add service annotation (or add service annotation in kafkabridge crd) about load balancer or should I create 1 more k8s loadbalancer service resource?

**Reply by scholzj** [reply]
For the Bridge, you can expose it your self. Either by creating some Ingress resource, or additional load-balancer service, or using some API gateway etc. Unlike Kafka the bridge is more simple to do this.
